includes/class-wc-form-handler.php

```diff
@@ -803,6 +809,31 @@ public static function add_to_cart_action( $url = false ) {
 			return;
 		}
 
+		$nonce_active = apply_filters( 'woocommerce_add_to_cart_nonce_active', false );
```

Comment:
Not keen on the naming of this hook. 

`woocommerce_add_to_cart_enable_nonce_check`? More descriptive IMO.